config/jobs/kubernetes/sig-release/release-branch-jobs/1.22.yaml

```diff
@@ -57,6 +57,9 @@ periodics:
       - --extract=ci/latest-1.22
       - --extract-ci-bucket=k8s-release-dev
       - --env=KUBE_CONTAINER_RUNTIME=docker
+      - --env=KUBE_NODE_OS_DISTRIBUTION=gci
+      - --env=KUBE_GCE_NODE_IMAGE=cos-85-13310-1041-9
```

Comment:
This is the same COS version we were using before https://github.com/kubernetes/test-infra/pull/27903